.editorconfig

```diff
@@ -3,8 +3,21 @@ root = true
 
 ; Unix-style newlines
 [*]
+charset = utf-8
 end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
 
 [*.php]
 indent_style = space
 indent_size = 4
+
+[*.{html,twig}]
```

Comment:
worth merging with php above?

Reply:
maybe but what happen if someone wants to add specific stuff later to only php ? or only to the html/twig ? I think is better to keep it separate to make clear that is not the same standard but I do not really mind

Reply:
that's future talk :) and if it's possible to add specific rules not an issue as well.

```
[*.{php, html, twig}]
global

[*.html]
specific
```

Also.. what about yaml, xml, json, ini?